Core/src/org/sleuthkit/autopsy/datamodel/Accounts.java

```diff
@@ -77,11 +78,16 @@
 public class Accounts extends Observable implements AutopsyVisitableItem {
```

Comment:
yes, to work in the tree it has to implement AutopsyVisitableItem but that interface is not public.